Lib/defcon/objects/layerSet.py

```diff
@@ -283,6 +283,8 @@ def save(self, writer, saveAs=False, progressBar=None):
                     newName = actionData["newName"]
                     if oldName in writer.layerContents:
                         writer.renameGlyphSet(oldName, newName)
+                        if newName == self.defaultLayer.name:
+                            writer.renameGlyphSet(newName, newName, defaultLayer=True)
```

Comment:
why do you have to call `renameGlyphSet` twice in the case of `defaultLayer`, with old and new names both being `newName`?

Reply:
I mean, wouldn't it also work if you did this?

```python
defaultLayer = newName == self.defaultLayer.name
writer.renameGlyphSet(oldName, newName, defaultLayer=defaultLayer)
```

(also this probably belongs to a separate PR, but it's ok)